examples/src/routes/BasicComponents.jsx

```diff
@@ -18,6 +18,7 @@ import HeatmapExample from "../components/HeatmapExample";
 import TableExample from "../components/TableExample";
 import HeadlineExample from "../components/HeadlineExample";
 import KpiExample from "../components/KpiExample";
+import GeoPushpinChartWithCategoryLegendExample from "../components/GeoPushpinChartConfigurationExample";
```

Comment:
import name is different with file's name

Reply:
fixed